src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Mashine[] mashines = {
```

Comment:
The array type `Mashine` is likely a typo. It should be `Machine` to match the corrected class name.